### PR TITLE
Adding CC* sites view

### PR DESCRIPTION
**Pull request by djw8605**
Waiting on:

- [x] Approval from Janet

### CODE REVIEW

**Comment by djw8605** [comment]
Can I get a courtesy review before merging.

File: src/templates/iframe.html.j2 [hunk 1]
```diff
@@ -239,14 +248,22 @@ function init(scrollwheel) {
             }
         }
 
+        var marker = new google.maps.Marker({
+            position: new google.maps.LatLng(sites[site][0], sites[site][1]),
+            map: map,
+            zIndex: resourceCounter,
+            icon: icon,
```
Comment (matyasselmeci) [inline src/templates/iframe.html.j2:255]: Is "icon" defined at this point?

**Review by matyasselmeci — APPROVED** [review]
One potential problem (see comment) but otherwise looks good so pre-approving.

File: src/templates/iframe.html.j2 (hunk 1 above)
Comment (djw8605) [reply src/templates/iframe.html.j2:255]: Good catch.  For some reason, it didn't complain.  Fixing.

**Pull request merged**